### PR TITLE
RODE-55 Adding initial version of grafeas-elasticsearch chart

### PR DESCRIPTION
**Pull request by jknight-liatrio**

### CODE REVIEW

File: charts/grafeas-elasticsearch/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+              protocol: TCP
+          volumeMounts:
+          - name: config-volume
+            mountPath: /etc/config
+{{- if eq .Values.certificates.enabled true }}
+          - name: certificates
+            mountPath: /certificates
+{{- end }}
+{{- if .Values.persistence.enabled }}
+{{- if eq .Values.persistence.storageType "embedded" }}
```
Comment (mrparkers) [inline charts/grafeas-elasticsearch/templates/deployment.yaml:49]: I don't think this is needed for our chart - we can probably assume that all users will be using the `elasticsearch` storage type.

File: charts/grafeas-elasticsearch/templates/configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        address: "0.0.0.0:{{ .Values.container.port }}"
+{{- if eq .Values.certificates.enabled true }}
+        # PKI configuration
+        cafile: /certificates/ca.crt
+        keyfile: /certificates/server.key
+        certfile: /certificates/server.crt
+{{- end }}
+        cors_allowed_origins:
+        # - "http://example.net"
+      storage_type: {{ .Values.grafeas.storageType }}
```
Comment (mrparkers) [inline charts/grafeas-elasticsearch/templates/configmap.yaml:20]: we can probably hardcode this to `elasticsearch`

**Comment by jknight-liatrio** [comment]
@mrparkers I think I agree with this. I've got another ticket to clean up this chart to make it work the way we need. I wanted to get something in here to get the helm release in here.

File: charts/grafeas-elasticsearch/templates/configmap.yaml (hunk 2 above)
Comment (jknight-liatrio) [reply charts/grafeas-elasticsearch/templates/configmap.yaml]: Removed.

File: charts/grafeas-elasticsearch/templates/deployment.yaml (hunk 1 above)
Comment (jknight-liatrio) [reply charts/grafeas-elasticsearch/templates/deployment.yaml]: Removed.

**Pull request merged**